packages/@aws-cdk/aws-apprunner-alpha/lib/service.ts

```diff
@@ -1074,6 +1147,10 @@ export class Service extends cdk.Resource implements iam.IGrantable {
       throw new Error('configurationValues cannot be provided if the ConfigurationSource is Repository');
     }
 
+    if (this.props.healthCheckConfiguration?.path && this.props.healthCheckConfiguration?.protocol !== HealthCheckProtocolType.HTTP) {
```

Comment:
Can you please also add validation for `HealthyThreshold`, `Interval`, `Timeout`, and `UnhealthyThreshold` min/max values according to [this](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-apprunner-service-healthcheckconfiguration.html#cfn-apprunner-service-healthcheckconfiguration-healthythreshold) spec?

Reply:
@lpizzinidev 

Thanks for review! I missed!

I changed them, please see again.

https://github.com/aws/aws-cdk/pull/27029/commits/2a11dc4f2d510a68f8e9be333ef911890e21553c